code/modules/power/supermatter/supermatter_delamination.dm

```diff
@@ -1,13 +1,101 @@
 /datum/supermatter_delamination
```

Comment:
what is the point of this being a datum exactly?

Reply:
is easier to expand on a separate datum rather than the supermatter object itself. and i like datums